packages/jsx2mp-runtime/scripts/platform-plugin.js

```diff
@@ -0,0 +1,123 @@
+const t = require('@babel/types');
+// Collector import specifiers
+const specified = [];
+const platformEnvValues = ['isMiniApp', 'isWeChatMiniProgram'];
+const platformMap = {
+  ali: 'isMiniApp',
+  wechat: 'isWeChatMiniProgram'
+};
+
+function variableDeclarationMethod(name, value) {
+  return t.variableDeclaration('const', [
+    t.variableDeclarator(t.identifier(name), t.booleanLiteral(value))
+  ]);
+}
+
+/**
+ * generator object expression
+ *
+ * @param  {string} platformName specified platform value it true
```

Comment:
specified platform value it true -> Set corresponding specified platform value to true